agent/src/test/java/com/teamscale/jacoco/agent/convert/ConverterTest.java

```diff
@@ -36,6 +36,24 @@ public void testSmokeTest(@TempDir File tempDir) throws Exception {
 		assertThat(xml).isNotEmpty().contains("<package").contains("<sourcefile").contains("<counter").contains("TestClass");
 	}
 
+	@Test
+	public void testNestedJar(@TempDir File tempDir) throws Exception {
+		File execFile = new File(getClass().getResource("coverage.exec").toURI());
+		File classFile = new File(getClass().getResource("TestClass.jar.zip").toURI());
+		File outputFile = new File(tempDir, "coverage.xml");
+
+		ConvertCommand arguments = new ConvertCommand();
+		arguments.inputFiles = Collections.singletonList(execFile.getAbsolutePath());
+		arguments.outputFile = outputFile.getAbsolutePath();
+		arguments.classDirectoriesOrZips = Collections.singletonList(classFile.getAbsolutePath());
+
+		new Converter(arguments).runJaCoCoReportGeneration();
+
+		String xml = FileSystemUtils.readFileUTF8(outputFile);
+		System.err.println(xml);
```

Comment:
This was not intended to be kept, right?

Reply:
Uhh, I just copy-pasted the other test above :joy: I will delete both occurrences.